src/fake_data/provide_hass.ts

```diff
@@ -203,6 +203,7 @@ export const provideHass = (
     translationMetadata: translationMetadata as any,
     dockedSidebar: "auto",
     vibrate: true,
+    suspendWhenHidden: true,
```

Comment:
Let's set this to false. There is no connection.
```suggestion
    suspendWhenHidden: false,
```